JavaScript and Ajax/DeleteStudentRecord.page

```diff
@@ -0,0 +1,45 @@
+<apex:page >
+    <h1>
+        Do you want to delete these records
+    </h1>
+    <body>
+        <form style="margin: 20px;" id="Confirmation">
+            <input type="button" onclick = "deleteRecords()" value="Confirm" />
+            <input type="button" onclick = "cancel()" value="cancel" />
```

Comment:
there is no `cancel` function in this page...

Reply:
this page have a cancel button also.

Reply:
function*